src/core/LoadingSpinner/LoadingSpinner.md

```diff
@@ -0,0 +1,95 @@
+```js
+import { LoadingSpinner } from 'suomifi-ui-components';
+
+<>
+  <div aria-live="assertive">
```

Comment:
Does this serve any purpose here as things presented inside it appear to be static?

Reply:
No. I will remove it.